scripts/lib/aws/sagemaker.sh

```diff
@@ -195,9 +197,150 @@ assert_aws_sagemaker_model_arn() {
   local model_response=$(daws sagemaker describe-model --model-name "$model_name" --region "$AWS_REGION")
   local aws_model_arn=$(echo "$model_response" | jq -r '.ModelArn')
   if [[ ( -z "$aws_model_arn" || "$expected_model_arn" != "$aws_model_arn" ) ]]; then
-    debug_msg "ERROR: ModelArn did not match/not found for $model_name"
+    debug_msg "[ERROR] ModelArn did not match/not found for $model_name"
     return 1
   fi
 
   return 0
-}
\ No newline at end of file
+}
+
+# is_aws_sagemaker_trainingjob_exists() returns 0 if an TrainingJob with the supplied name
+# exists, 1 otherwise.
+#
+# training_job_created TRAINING_JOB_NAME
+# Arguments:
+#
+#   TRAINING_JOB_NAME  required string for the name of the bucket to check
+#
+# Usage:
+#
+#   if ! training_job_created "$training_job_name"; then
+#       echo "Training Job $training_job_name does not exist!"
+#   fi
+is_aws_sagemaker_trainingjob_exists() {
```

Comment:
I think "aws" is redundant :) Also, to match functions in other `scripts/lib/aws` modules, might be good to call this `sagemaker_trainingjob_exists()`. For example, see the [`sns_topic_exists()` function](https://github.com/aws/aws-controllers-k8s/blob/cc3c433f3407df06554cd7d937f7e0286b33bc59/scripts/lib/aws/sns.sh#L26).